components/ProgressView/src/MDCProgressView.h

```diff
@@ -86,4 +86,20 @@ IB_DESIGNABLE
          animated:(BOOL)animated
        completion:(void (^__nullable)(BOOL finished))completion;
 
+/**
+ Start the progress bar's indeterminate animation.
+ */
+- (void)startAnimating;
+
+/**
+ Stop the progress bar's indeterminate animation.
+ */
+- (void)stopAnimating;
```

Comment:
Does this API stop animating immediately or does it allow the animation to "finish" nicely?  If it allows the animation to "finish", please provide an API like `- (void)stopAnimatingAnimated:(BOOL) completion:(**block**)`

Reply:
MDCActivityIndicator's `stopAnimating` API allows the animation to finish gracefully so we should minimally meet that same API contract here. a `- (void)stopAnimatingAnimated:(BOOL) completion:(**block**)` method would be a good addition as well for clarity's sake.